runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineWorkerPoolOptions.java

```diff
@@ -204,20 +204,18 @@ public String create(PipelineOptions options) {
   List<String> getFilesToStage();
   void setFilesToStage(List<String> value);
 
-  String getWorkerDiskType();
-
   /**
-   * Specifies what type of persistent disk should be used. The value should be a
-   * full disk type resource, e.g.,
-   * compute.googleapis.com/projects//zones//diskTypes/pd-ssd. For more information,
+   * Specifies what type of persistent disk is used. The value is a full disk type resource,
+   * e.g., compute.googleapis.com/projects//zones//diskTypes/pd-ssd. For more information,
    * see the <a href="https://cloud.google.com/compute/docs/reference/latest/diskTypes">API
    * reference documentation for DiskTypes</a>.
    */
-  @Description("Specifies what type of persistent disk should be used. The "
-      + "value should be a full URL of a disk type resource, e.g., "
+  @Description("Specifies what type of persistent disk is used. The "
+      + "value is a full URL of a disk type resource, e.g., "
```

Comment:
I see the way you're changing this now. I think the javadoc changes are good (as those are used by someone looking at the getter javadoc), but the Description changes should be reverted (as those are used to specify help for pipeline options, typically on the command line.)

cc: @lukecwik 

Sub, you are clearly thinking about this with finer detail than we have. Thanks.